psamm/datasource/sbml.py

```diff
@@ -872,12 +894,20 @@ def write_model(self, file, model):
             species_tag.set(
                 self._sbml_tag('compartment'),
                 model_compartments[species.compartment])
+            species_tag.set(self._sbml_tag('constant'), 'true')
+            species_tag.set(self._sbml_tag('boundaryCondition'), 'true')
```

Comment:
It said that if constant was true, boundary condition also had to be true (in that validator).